src/server/models/career.model.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+let careerSchema = new Schema({
```

Comment:
What is a "career"? A job position? Then it should probably be called "position".

https://developer.linkedin.com/docs/fields/basic-profile